templates/common/aws/units/afterburn-hostname.service.yaml

```diff
@@ -0,0 +1,17 @@
+name: afterburn-hostname.service
```

Comment:
I thought this was being covered in a separate PR? We should keep the scope of this PR just to the feature gate changes and have a separate PR for the AWS hostname issue

Reply:
Yes it was, although this implementation is the only one which worked. An issue is still there in the other PR and there is no way to fix it: https://github.com/openshift/machine-config-operator/pull/2401#issuecomment-789739611

The implementation does not belong here, was a temporary fix for testing. Removed the last commit.